ruby/ql/src/change-notes/2013-05-02-post-unicode-normalization-query.md

```diff
@@ -0,0 +1,4 @@
+---
+category: newQuery
+---
+* Added a new query, `ruby/post-unicode-normalization`, to detect a misuse of a post-unicode normalization.
```

Comment:
I suggest the change note is deleted completely until the query is promoted to be part of the standard set of queries. Otherwise it's will be misleading.

Reply:
Done via [69ca49f](https://github.com/github/codeql/pull/12992/commits/69ca49f168e13afb3d7cc07befbdb7db9bf411a3).